engine/crates/integration-tests/tests/postgresql/introspection.rs

```diff
@@ -851,6 +1067,23 @@ fn two_schemas_same_table_name() {
           pageInfo: PageInfo!
         }
 
+        type PublicUserDelete {
+          id: Int!
+        }
+
+        input PublicUserDeleteCollection {
+          id: IntSearchFilterInput
+          """
+            All of the filters must match
+          """ ALL: [PublicUserDeleteCollection]
+          """
+            None of the filters must match
+          """ NONE: [PublicUserDeleteCollection]
+          """
+            At least one of the filters must match
+          """ ANY: [PublicUserDeleteCollection]
```

Comment:
I don't know how important this is but formatting is weird in these types, I think we're missing a newline after the `"""`.

Reply:
yeah, it's the formatter that does that. not an issue, and not happening on production